.github/workflows/ci.yml

```diff
@@ -3,6 +3,19 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
should we specify a python version here? I assume that could affect the lint results.

I don't know what the minimum python version supported by ros 2 is, but I assume `3.8` is a safe choice since that's what comes with ubuntu 20.04 (used by foxy and galatic)

Reply:
Don't we want to know (before our users do) that Python 3.10 breaks us?

Reply:
Good question. I'm not sure what is common, but I would think that most people are using the version of python that comes with ubuntu (since ROS requires a specific ubuntu distribution)?

If we really care about that then it would be best to test both the minimum and maximum versions. But for lint it's probably safer to run on the minimum supported version?

Open to ideas.